tests/isolated/protobuf_test.py

```
@@ -0,0 +1,35 @@
+MODULES = [
+    "fiftyone.core.cli",
+    "fiftyone.server.routes",
+    "fiftyone.utils.data.importers",
+]
+
+
+def test_incompatible_protobuf():
+    subprocess.check_call(
+        [sys.executable, "-m", "pip", "install", "protobuf==4.25.9"]
+    )
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)